src/formatting.ts

```diff
@@ -32,6 +33,13 @@ function format(
         result += record.msg;
       } else if (value === level) {
         result += levelToString(record.level);
+      } else if (value === trace) {
+        const errorStack = new Error().stack ?? '';
+        const splitErrorStack = errorStack.split('\n');
+        const position =
+          splitErrorStack.findIndex((value) => /Logger\.log/.test(value)) ?? 0;
+        const formattedStack = splitErrorStack.splice(position + 1).join('\n');
```

Comment:
Can you show us an example of this log message? I'd like to get rid of the first 1 or 2 lines since the stack should really start where the `logger.info()` is made, and not start here.

Reply:
Here is an example
```
2020-01-01T00:00:00.000Z:DEBUG MESSAGE
    at Logger.debug (/home/faulkes/matrixcode/polykey/js-logger/src/Logger.ts:78:10)
    at Object.<anonymous> (/home/faulkes/matrixcode/polykey/js-logger/tests/index.test.ts:111:12)
    at Promise.then.completed (/home/faulkes/matrixcode/polykey/js-logger/node_modules/jest-circus/build/utils.js:391:28)
    at new Promise (<anonymous>)
    at callAsyncCircusFn (/home/faulkes/matrixcode/polykey/js-logger/node_modules/jest-circus/build/utils.js:316:10)"
```

The line pointing to where the message is is the 3rd line containing `at Object.<anonymous> (/home/faulkes/matrixcode/polykey/js-logger/tests/index.test.ts:111:12)`. I kept the line `Logger.debug` here to show what kind of debug message it was but we can skip that one as well.

Reply:
Yea I think the `Logger.debug` is not necessary, since we already know this simply through the `${level}` symbol. So we should start with just:

```
    at Object.<anonymous> (/home/faulkes/matrixcode/polykey/js-logger/tests/index.test.ts:111:12)
```

Reply:
Fixed.